clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/PersistenceDAO.scala

```diff
@@ -124,7 +123,7 @@ trait PersistenceDAO extends LazyLogging {
        * We use `mapAsync` then `mapConcat`, instead of just `flatMapConcat`,
        * so we can sort the intermediate Seq result of the ls.
        */
-      .mapAsync(1)(Directory.ls(_).runWith(Sink.seq))
+      .mapAsync(recoveryParallelism)(Directory.ls(_).runWith(Sink.seq))
```

Comment:
I didn't think that this operation was implicated in the profiling results. Or were we not able to distinguish it from the `Files.readAllBytes()` below?

Reply:
Yeah, they both hit the network. `Files.readAllBytes` is the one I'd expect the most benefit from parallelizing, but we should be able to squeeze out some wait time by doing the same thing here.